### PR TITLE
Redirecting

### LINKED ISSUE #1: [Hui] Add redirect to "save draft" and "back"

### PR DESCRIPTION
**Pull request by HuiFang56**
Redirecting Back

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,4 @@
 - [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
 - [sort-imports](https://marketplace.visualstudio.com/items?itemName=amatiasq.sort-imports)
 - [Flow Language Support](https://marketplace.visualstudio.com/items?itemName=flowtype.flow-for-vscode)
+h
```
Comment (can-z) [inline README.md]: what's this? Remove if not needed

File: client/src/components/PickupDetail.js [hunk 2]
```diff
@@ -44,10 +50,10 @@ const PickupDetail: () => React$Node = () => {
         </div>
       </div>
       <div class="container-fluid m-1">
-        <a role="button" href="/add-pickup" className="btn btn-primary mx-1">
+        <a role="button" href="/" className="btn btn-primary mx-1">
```
Comment (can-z) [inline client/src/components/PickupDetail.js]: Why do these two buttons have different ways of redirecting? Use the same one for both.

File: client/src/components/PickupDetail.js [hunk 3]
```diff
@@ -44,10 +50,10 @@ const PickupDetail: () => React$Node = () => {
         </div>
       </div>
       <div class="container-fluid m-1">
-        <a role="button" href="/add-pickup" className="btn btn-primary mx-1">
+        <a role="button" onClick={backToLanding} className="btn btn-primary mx-1">
```
Comment (can-z) [inline client/src/components/PickupDetail.js:53]: what's the advantage of using history than `href`? seems to me `href` is simpler.

Comment (HuiFang56) [reply client/src/components/PickupDetail.js:53]: From my understanding, by history, everytime when we direct to a page we already loaded once, our browser does not need to reload it again, as the page was just hidden somewhere for further use.

**Review by can-z — APPROVED** [review]

**Pull request merged**